After POD csi-rbdplugin Evicted - POD - CrashLoopBackOff

**Describe the bug**
After POD csi-rbdplugin - Evicted 
Pod not restarting, status - CrashLoopBackOff

**Kubernetes and Ceph CSI Versions**
Kubernetes: 1.13
Ceph CSI: 1.0.0

**Ceph CSI Driver logs**

-c csi-rbdplugin                                                                                                                                                              
```
/rbdplugin flag redefined: log_dir
panic: /rbdplugin flag redefined: log_dir

goroutine 1 [running]:
flag.(*FlagSet).Var(0xc00008a180, 0x13b6e60, 0x1e339b0, 0x1268e66, 0x7, 0x1295103, 0x2f)
	/home/travis/.gimme/versions/go1.11.5.linux.amd64/src/flag/flag.go:805 +0x529
flag.(*FlagSet).StringVar(0xc00008a180, 0x1e339b0, 0x1268e66, 0x7, 0x0, 0x0, 0x1295103, 0x2f)
	/home/travis/.gimme/versions/go1.11.5.linux.amd64/src/flag/flag.go:708 +0x8a
github.com/ceph/ceph-csi/vendor/k8s.io/klog.InitFlags(0xc00008a180)
	/home/travis/gopath/src/github.com/ceph/ceph-csi/vendor/k8s.io/klog/klog.go:411 +0x7b
main.main()
	/home/travis/gopath/src/github.com/ceph/ceph-csi/cmd/rbd/main.go:45 +0x3c
```
-c driver-registrar
```
I0205 14:32:30.913425 1494627 main.go:138] Version: v1.0.0-rc2-7-g9952449f
I0205 14:32:30.913497 1494627 main.go:145] Attempting to open a gRPC connection with: "/var/lib/kubelet/plugins_registry/csi-rbdplugin/csi.sock"
I0205 14:32:30.913520 1494627 connection.go:68] Connecting to /var/lib/kubelet/plugins_registry/csi-rbdplugin/csi.sock
I0205 14:32:30.913804 1494627 connection.go:95] Still trying, connection is CONNECTING
I0205 14:32:30.913981 1494627 connection.go:95] Still trying, connection is TRANSIENT_FAILURE
I0205 14:32:31.914194 1494627 connection.go:95] Still trying, connection is CONNECTING
I0205 14:32:31.914610 1494627 connection.go:95] Still trying, connection is TRANSIENT_FAILURE
I0205 14:32:32.956180 1494627 connection.go:95] Still trying, connection is CONNECTING
I0205 14:32:32.956592 1494627 connection.go:95] Still trying, connection is TRANSIENT_FAILURE
I0205 14:32:34.132867 1494627 connection.go:95] Still trying, connection is CONNECTING
I0205 14:32:35.198675 1494627 connection.go:95] Still trying, connection is TRANSIENT_FAILURE
I0205 14:32:36.173900 1494627 connection.go:95] Still trying, connection is TRANSIENT_FAILURE
I0205 14:32:37.143832 1494627 connection.go:95] Still trying, connection is TRANSIENT_FAILURE
I0205 14:32:38.218737 1494627 connection.go:95] Still trying, connection is TRANSIENT_FAILURE
I0205 14:32:39.045109 1494627 connection.go:95] Still trying, connection is TRANSIENT_FAILURE
I0205 14:32:39.907861 1494627 connection.go:95] Still trying, connection is TRANSIENT_FAILURE
I0205 14:32:40.746752 1494627 connection.go:95] Still trying, connection is TRANSIENT_FAILURE
I0205 14:32:41.667274 1494627 connection.go:95] Still trying, connection is TRANSIENT_FAILURE
I0205 14:32:42.673429 1494627 connection.go:95] Still trying, connection is TRANSIENT_FAILURE
I0205 14:32:43.799053 1494627 connection.go:95] Still trying, connection is TRANSIENT_FAILURE
I0205 14:32:44.684833 1494627 connection.go:95] Still trying, connection is TRANSIENT_FAILURE
I0205 14:32:45.637260 1494627 connection.go:95] Still trying, connection is TRANSIENT_FAILURE
I0205 14:32:46.564725 1494627 connection.go:95] Still trying, connection is TRANSIENT_FAILURE
I0205 14:32:47.552383 1494627 connection.go:95] Still trying, connection is TRANSIENT_FAILURE
I0205 14:32:48.465682 1494627 connection.go:95] Still trying, connection is TRANSIENT_FAILURE
I0205 14:32:49.383057 1494627 connection.go:95] Still trying, connection is TRANSIENT_FAILURE
I0205 14:32:50.454832 1494627 connection.go:95] Still trying, connection is TRANSIENT_FAILURE
I0205 14:32:51.342399 1494627 connection.go:95] Still trying, connection is TRANSIENT_FAILURE
I0205 14:32:52.223837 1494627 connection.go:95] Still trying, connection is TRANSIENT_FAILURE
I0205 14:32:53.168333 1494627 connection.go:95] Still trying, connection is TRANSIENT_FAILURE
I0205 14:32:54.196648 1494627 connection.go:95] Still trying, connection is CONNECTING
I0205 14:32:54.196808 1494627 connection.go:95] Still trying, connection is TRANSIENT_FAILURE
I0205 14:32:55.341920 1494627 connection.go:95] Still trying, connection is TRANSIENT_FAILURE
I0205 14:32:56.259305 1494627 connection.go:95] Still trying, connection is TRANSIENT_FAILURE
I0205 14:32:57.178270 1494627 connection.go:95] Still trying, connection is TRANSIENT_FAILURE
I0205 14:32:58.279355 1494627 connection.go:95] Still trying, connection is CONNECTING
I0205 14:32:58.279488 1494627 connection.go:95] Still trying, connection is TRANSIENT_FAILURE
I0205 14:32:59.162388 1494627 connection.go:95] Still trying, connection is TRANSIENT_FAILURE
I0205 14:33:00.308616 1494627 connection.go:95] Still trying, connection is TRANSIENT_FAILURE
I0205 14:33:01.387372 1494627 connection.go:95] Still trying, connection is CONNECTING
I0205 14:33:01.387569 1494627 connection.go:95] Still trying, connection is TRANSIENT_FAILURE
I0205 14:33:02.397185 1494627 connection.go:95] Still trying, connection is TRANSIENT_FAILURE
I0205 14:33:03.208750 1494627 connection.go:95] Still trying, connection is TRANSIENT_FAILURE
I0205 14:33:04.072268 1494627 connection.go:95] Still trying, connection is TRANSIENT_FAILURE
I0205 14:33:05.115113 1494627 connection.go:95] Still trying, connection is CONNECTING
I0205 14:33:05.115269 1494627 connection.go:95] Still trying, connection is TRANSIENT_FAILURE
I0205 14:33:06.305446 1494627 connection.go:95] Still trying, connection is TRANSIENT_FAILURE
I0205 14:33:07.137345 1494627 connection.go:95] Still trying, connection is TRANSIENT_FAILURE
I0205 14:33:08.175457 1494627 connection.go:95] Still trying, connection is TRANSIENT_FAILURE
I0205 14:33:08.999219 1494627 connection.go:95] Still trying, connection is TRANSIENT_FAILURE
I0205 14:33:10.076177 1494627 connection.go:95] Still trying, connection is TRANSIENT_FAILURE
I0205 14:33:10.997058 1494627 connection.go:95] Still trying, connection is CONNECTING
I0205 14:33:10.997263 1494627 connection.go:95] Still trying, connection is TRANSIENT_FAILURE
I0205 14:33:11.866409 1494627 connection.go:95] Still trying, connection is TRANSIENT_FAILURE
I0205 14:33:12.882878 1494627 connection.go:95] Still trying, connection is CONNECTING
I0205 14:33:13.900774 1494627 connection.go:95] Still trying, connection is TRANSIENT_FAILURE
I0205 14:33:14.812334 1494627 connection.go:95] Still trying, connection is TRANSIENT_FAILURE
I0205 14:33:15.781761 1494627 connection.go:95] Still trying, connection is TRANSIENT_FAILURE
I0205 14:33:16.793997 1494627 connection.go:95] Still trying, connection is CONNECTING
I0205 14:33:16.794145 1494627 connection.go:95] Still trying, connection is TRANSIENT_FAILURE
I0205 14:33:17.695865 1494627 connection.go:95] Still trying, connection is TRANSIENT_FAILURE
I0205 14:33:18.608759 1494627 connection.go:95] Still trying, connection is CONNECTING
I0205 14:33:18.608924 1494627 connection.go:95] Still trying, connection is TRANSIENT_FAILURE
I0205 14:33:19.724235 1494627 connection.go:95] Still trying, connection is CONNECTING
I0205 14:33:19.724374 1494627 connection.go:95] Still trying, connection is TRANSIENT_FAILURE
I0205 14:33:20.669114 1494627 connection.go:95] Still trying, connection is CONNECTING
I0205 14:33:20.669245 1494627 connection.go:95] Still trying, connection is TRANSIENT_FAILURE
I0205 14:33:21.821516 1494627 connection.go:95] Still trying, connection is CONNECTING
I0205 14:33:21.821693 1494627 connection.go:95] Still trying, connection is TRANSIENT_FAILURE
I0205 14:33:22.740681 1494627 connection.go:95] Still trying, connection is CONNECTING
I0205 14:33:22.740855 1494627 connection.go:95] Still trying, connection is TRANSIENT_FAILURE
I0205 14:33:23.898478 1494627 connection.go:95] Still trying, connection is CONNECTING
I0205 14:33:23.898626 1494627 connection.go:95] Still trying, connection is TRANSIENT_FAILURE
I0205 14:33:24.737614 1494627 connection.go:95] Still trying, connection is CONNECTING
I0205 14:33:24.737738 1494627 connection.go:95] Still trying, connection is TRANSIENT_FAILURE
I0205 14:33:25.928597 1494627 connection.go:95] Still trying, connection is CONNECTING
I0205 14:33:25.928699 1494627 connection.go:95] Still trying, connection is TRANSIENT_FAILURE
I0205 14:33:26.758419 1494627 connection.go:95] Still trying, connection is CONNECTING
I0205 14:33:26.758507 1494627 connection.go:95] Still trying, connection is TRANSIENT_FAILURE
I0205 14:33:27.647493 1494627 connection.go:95] Still trying, connection is CONNECTING
I0205 14:33:27.647651 1494627 connection.go:95] Still trying, connection is TRANSIENT_FAILURE
I0205 14:33:28.720068 1494627 connection.go:95] Still trying, connection is CONNECTING
I0205 14:33:28.720205 1494627 connection.go:95] Still trying, connection is TRANSIENT_FAILURE
I0205 14:33:29.616829 1494627 connection.go:95] Still trying, connection is CONNECTING
I0205 14:33:29.616924 1494627 connection.go:95] Still trying, connection is TRANSIENT_FAILURE
I0205 14:33:30.541616 1494627 connection.go:95] Still trying, connection is CONNECTING
I0205 14:33:30.542145 1494627 connection.go:95] Still trying, connection is TRANSIENT_FAILURE
I0205 14:33:30.914045 1494627 connection.go:88] Connection timed out
I0205 14:33:30.914075 1494627 main.go:153] Calling CSI driver to discover driver name.
I0205 14:33:30.914108 1494627 connection.go:136] GRPC call: /csi.v1.Identity/GetPluginInfo
I0205 14:33:30.914116 1494627 connection.go:137] GRPC request:
I0205 14:33:30.914225 1494627 connection.go:139] GRPC response:
I0205 14:33:30.914277 1494627 connection.go:140] GRPC error: rpc error: code = Unavailable desc = all SubConns are in TransientFailure
E0205 14:33:30.914291 1494627 main.go:158] rpc error: code = Unavailable desc = all SubConns are in TransientFailure
```

--- a/cmd/cephfs/main.go
+++ b/cmd/cephfs/main.go
@@ -42,7 +42,6 @@ var (
 )
 
 func main() {
-	klog.InitFlags(nil)
 	flag.Parse()
 
 	if err := createPersistentStorage(path.Join(cephfs.PluginFolder, "controller")); err != nil {

--- a/cmd/rbd/main.go
+++ b/cmd/rbd/main.go
@@ -42,7 +42,6 @@ var (
 )
 
 func main() {
-	klog.InitFlags(nil)
 	flag.Parse()
 
 	if err := createPersistentStorage(path.Join(rbd.PluginFolder, "controller")); err != nil {

